test/integration/associations/belongs-to.test.js

```diff
@@ -872,6 +883,53 @@ describe(Support.getTestDialectTeaser('BelongsTo'), () => {
               return Task.findOne({ where: { title: 'some task' } }).then(foundTask => {
                 return foundTask.getUser().then(foundUser => {
                   expect(foundUser.username).to.equal('bob');
+                  return this.sequelize.getQueryInterface().getForeignKeyReferencesForTable('Tasks').then(foreignKeysDescriptions => {
+                    expect(foreignKeysDescriptions[0]).to.have.property('referencedColumnName', 'the_user_name_field');
```

Comment:
Please reduce promise nesting and use https://www.chaijs.com/api/bdd/#includeval-msg